docs/proposals/The proposal about building a high frequency Sedna based end to end use case in ModelBox format.md

```diff
@@ -0,0 +1,94 @@
+[toc]
+
+# The proposal about building a high frequency Sedna based end to end use case in ModelBox format
+
+## 1. Background
+
+> Sedna is an edge cloud collaborative AI project at KubeEdge SIG AI. Benefiting from the edge-cloud collaboration capabilities provided by KubeEdge, Sedna can realize cross-edge-cloud collaborative training and collaborative reasoning capabilities, and Sedna supports a variety of popular AI frameworks. **Sedna can simply enable edge-cloud synergy for existing training and inference scripts**, bringing the benefits of lower costs, improved model performance, and data privacy. **The goal of ModelBox is to solve the programming complexity of AI developers when developing AI applications**, reduce the development difficulty of AI applications, and hand over complex data processing to ModelBox. Developers mainly focus on the business logic itself, not the software details. While improving the efficiency of AI inference development, it ensures the performance, reliability, security and other attributes of the software, **so as to better enable Sedna developers to manage and use the Sedna framework**.
+
+## 2. Motivation
+
+### Goals
+
+- ModelBox provides visual layout function to help Sedna developers improve the programming complexity when developing edge cloud collaborative AI.
```

Comment:
The fcontefntf here is ok, but the description could be modified to make it look like the goals of Sedna, instead of ModelBox.

Reply:
The contest is modified.